tests/test_integrations/test_nessus.py

```
@@ -0,0 +1,49 @@
+
+
+class TestNessusIntegration:
+    def test_schema_validation(self):
+        schema = NessusService.get_schema()
+        assert schema is NessusIntegration
+
+        # Should raise error if required fields are missing
+        # url has default, but access_key and secret_key are required
+        try:
+            NessusIntegration(url="https://test:8834")
+            assert False, "Should have raised validation error for missing keys"
```

Ruff (0.15.18)

[warning] Assertion always fails, replace with `pytest.fail()`

(PT015)